InvalidPath.h

```diff
@@ -0,0 +1,27 @@
+#ifndef TICTACTOE_INVALIDPATH_H
+#define TICTACTOE_INVALIDPATH_H
+
+#include <iostream>
+#include <chrono>
+#include <thread>
+#include <Windows.h>
+#include <vector>
+
+using namespace std::chrono_literals;
+
+class Timeout {

```

Comment:
This name leads me to suggest that this will be a function object where, in fact, the object here is not